### PR TITLE
Nc news ticket#21

### PR DESCRIPTION
**Pull request by reece-b4**

### CODE REVIEW

**Comment by cjplatten** [comment]
Looks good!

File: __tests__/app.test.js [hunk 1]
```diff
@@ -133,6 +133,26 @@ describe("/api/articles/:article_id", () => {
   });
 })
 
+//ticket 21
+describe('/api/users', () =>{
+  describe('GET', () => {
+    test.only('200 - returns array of objects with username property', () => {
+      return request(app).get('/api/users')
+      .then((res)=>{
+        console.log(res.body, 'res.body')
```
Comment (cjplatten) [inline __tests__/app.test.js:142]: You're missing the status expect in this test.

Also - Console.log!

**Review by cjplatten — COMMENTED** [review]
Just one comment to address. (please ignore my previous comment)

File: __tests__/app.test.js (hunk 1 above)
Comment (reece-b4) [reply __tests__/app.test.js:142]: thanks, fixed

**Pull request merged**